cloudant-client/src/main/java/com/cloudant/client/api/Database.java

```diff
@@ -1334,6 +1393,21 @@ public DbInfo info() {
                 DbInfo.class);
     }
 
+    /**
+     * Get information about this database partition.
```

Comment:
I think this should say `Get information about a partition in this database` since you have to instruct the method which partition and also `this` has meaning in Java that could be misconstrued otherwise.